app/common/components/kendo-grid/editable-cell/get-action-cell.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+
+import { GridCell, GridCellProps } from '@progress/kendo-react-grid';
+
+export type ActionsComponent = React.ComponentType<GridCellProps>;
+
+interface GetActionCellParams {
+    actions?: ActionsComponent;
+}
+
+export function getActionCell({ actions: Actions }: GetActionCellParams) {
+    return class extends GridCell {
+        render() {
+            if (this.props.rowType !== "data") {
```

Comment:
It won't work correctly with group footer rowType.

Reply:
set `groupable={{ footer: 'visible' }}` for KendoGrid to look on issue.

Reply:
@xarety 
I fixed an issue for groupFooter ([74b5c5c](https://github.com/andreytrofimov/react-onboarding-practice-course/pull/7/commits/74b5c5c)), but there is another in groupHeader 

![image](https://user-images.githubusercontent.com/8394304/58896056-e5195000-86fd-11e9-8632-72a3242e18e7.png)

Reply:
Should we have a checkbox here?

Reply:
Yup, this issue in the SelectColumnCell component, you can fix it in your branch if you want (same fix as you did for actions cell). It was fixed in the cactus release branch, a bit later we'll update the task template repository.